cmd/cluster/context.go

```diff
@@ -110,14 +109,17 @@ func newCmdContext() *cobra.Command {
 		Short:             "Shows the context of a specified cluster",
 		Args:              cobra.ExactArgs(1),
 		DisableAutoGenTag: true,
-		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(ops.complete(cmd, args))
-			cmdutil.CheckErr(ops.run())
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := ops.setup(args)
+			if err != nil {
+				return err
+			}
+
+			return ops.run()
 		},
 	}
 
 	contextCmd.Flags().StringVarP(&ops.output, "output", "o", "long", "Valid formats are ['long', 'short', 'json']. Output is set to 'long' by default")
-	contextCmd.Flags().StringVarP(&ops.clusterID, "cluster-id", "C", "", "Cluster ID")
```

Comment:
In the longer term I want to standardize across all commands either positional arg for cluster or `--cluster-id` flag, not both.